document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/render/RenderDocumentReponse.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.document.generator.core.document.render;
+
+public class RenderDocumentReponse {
+
+    private DocumentGenerationCompleted generatedDocument;
```

Comment:
This class does not exist in the project nor its dependency in the pom.xml

Reply:
It is present in https://github.com/companieshouse/document-generator/pull/6 which needs to be PRd first. I should have held the PR off on this one.